tools/src/commands/GenerateBareDiffs.ts

```diff
@@ -0,0 +1,117 @@
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+import { PromisyClass, TaskQueue } from 'cwait';
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+
+import { EXPO_DIR } from '../Constants';
+import logger from '../Logger';
+
+function allVersions(min, max) {
+  const versions: string[] = [];
+  for (let i = min; i <= max; i++) {
+    versions.push(i.toString());
+  }
+  return versions;
+}
+
+function getReleasedSdkVersionRange() {
+  const packageJson = fs.readJSONSync(path.join(EXPO_DIR, 'packages', 'expo', 'package.json'));
+  const expoPackageVersion = packageJson.version;
+  let maxSdk = parseInt(expoPackageVersion.split('.')[0], 10);
+  // if current version is an alpha or beta, then one version behind is the latest stable release
+  if (expoPackageVersion.includes('alpha') || expoPackageVersion.includes('beta')) {
+    maxSdk--;
+  }
+  return {
+    maxSdk,
+    minSdk: maxSdk - 6,
+  };
+}
+
+const executeDiffCommand = async (diffDirPath: string, sdkFrom: string, sdkTo: string) => {
```

Comment:
Let's prefer using a function instead of an arrow function if we don't need to capture `this`.
```suggestion
async function executeDiffCommand(diffDirPath: string, sdkFrom: string, sdkTo: string) {
```